parts/AddRegistryForWS2019.cmd

```diff
@@ -0,0 +1,6 @@
+reg query HKLM\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters /v SGX_Launch_Config_Optin

```

Comment:
Is `Optin` supposed to be `Option`, or is it like "Opt In"? I imagine this is correct, I just have never used this registry setting before

Reply:
It is Optin (referring to opting in) registry setting.